django/core/management/commands/showmigrations.py

```diff
@@ -26,7 +26,11 @@ def add_arguments(self, parser):
         )
         formats.add_argument(
             '--plan', '-p', action='store_const', dest='format', const='plan',
-            help='Shows all migrations in the order they will be applied.',
+            help=(
+                'Shows all migrations in the order they will be applied. '
```

Comment:
Yeah, I agree. Taking out this file as a separate commit or PR sounds like a plan.